sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/Application.java

```diff
@@ -1,15 +1,35 @@
 package com.sap.ai.sdk.app;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sap.ai.sdk.orchestration.OrchestrationJacksonConfiguration;
+import javax.annotation.Nonnull;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.web.servlet.ServletComponentScan;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Primary;
 
 /** Main class to start the Spring Boot application. */
 @SpringBootApplication
 @ComponentScan({"com.sap.cloud.sdk", "com.sap.ai.sdk.app"})
 @ServletComponentScan({"com.sap.cloud.sdk", "com.sap.ai.sdk.app"})
 public class Application {
+
+  /**
+   * Temporary workaround to fix the issue with the Orchestration spec.
```

Comment:
(Question)

1. How is it temporary? (We are not planning to fix it. Matthias urged that AI SDK  classes shall never be used for front-end serializing, normally)
2. Is Spring picking this object mapper as default serializer? If true, nice!

Reply:
I thought the Orchestration spec would include discriminator fields that would allow us to remove this mapper at some point.